### DISCUSSION TITLE
Copy formatted text & paste in editor is removing text style

### DISCUSSION THREAD

**Discussion by ajaykachhawa** [singerdmx/flutter-quill#721] [General]
I have checked this on [Web] but it may be in [Mobile] & [Desktop] also with version flutter_quill: ^4.0.4

Copy formatted text & paste in editor is removing text style.

1. Select some style Bold/Italic/Any other
2. Cope that text & paste in new line
3. It is removing copied text style & pasting only normal text

AC - It should maintain text style after copy & paste.

**Comment by tschiekdev** [comment]
I would also be happy if there is a solution for this. Maybe with a separate button in the toolbar?

**Comment by Ibtesam-Mahmood** [comment]
This issue is not solved on version ^7.0.1
